core/decorators.py

```diff
@@ -4,6 +4,12 @@
 from discord.ext import commands
 
 
+async def ignore(coro):
```

Comment:
Not a big fan of except all clauses, but make it marginally better with `except Exception:` for PEP 8. Since this isn't really a decorator, maybe move it to utils.py?